eras/shelley/test-suite/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Version history for `cardano-ledger-shelley-test`
 
+## 1.5.0.0
+
+* Remove `myDiscard`
+* Require `cardano-ledger-core` >= 1.15
+
 ## 1.4.1.0
 
 *
```

Comment:
Ditto. Except in this case we do need to bump up the version.
Also we do not list changes to bounds in the changelog, those are no user visible changes.
```suggestion
## 1.5.0.0

* Remove `myDiscard`
```